napari/components/menu.py

```diff
@@ -0,0 +1,24 @@
+from typing import Callable, List, Optional
+
+from ..utils.events import EventedModel
+
+
+class MenuItem(EventedModel):
+    label: str
+    id: str
```

Comment:
In https://github.com/napari/npe2/pull/161 we have `key`  instead of `id`, we should probably make them match

Reply:
as discussed offline, id has a different meaning/intention than the original key